cumulusci.yml

```diff
@@ -32,6 +32,18 @@ tasks:
             output: robot/OutboundFunds/doc/OutboundFunds_tests.html
             
 flows:
+
+    deploy_unmanaged:
+        steps:
+            4: 
+                task: uninstall_packaged_incremental
+                options:
+                    # Ignore CustomObjectTranslation for localization testing
+                    ignore_types:
+                        - RecordType
+                        - CustomObjectTranslation
```

Comment:
This is the only part I don't think I follow. I don't believe we've had to suppress this in any context before. Can you share more about the workflow that requires us to do this?

Reply:
If we don't have this, we get errors during this task.  Will get a screen shot.

Reply:
Oh, is it throwing an error saying that you can't delete translation records corresponding to standard objects?

Reply:
Removing this.

We only need this when testing pseudo-localization with translation files.

Reply:
You'll see the error again, just so you know, but I can assist in fixing it - it happens after deployment into packaging.

Reply:
we when try pseduo-loc testing we get the following error: [https://gist.github.com/04f3ef46b18f870a9d62ea96991f00e5](https://gist.github.com/04f3ef46b18f870a9d62ea96991f00e5)